README.md

```diff
@@ -235,3 +235,4 @@ appear in the text.
         - text &rightarrow; `"cialis vs cialis spam guestbook.php?action=http://cialiswalmart.shop"`
     - **Output:** `3`
 
+
```

Comment:
Why extra whitespace?

Reply:
this is your addition

Reply:
want me to remove it?

Reply:
Right, sorry. Please go ahead and remove.